README.md

```diff
@@ -9,11 +9,23 @@ client-id: ${CLIENT_ID}
 client-secret: ${CLIENT_SECRET}
 project-key: ${PROJECT_KEY}
 ```
+## Docker set up (sbt-native-packager)
+Create a local docker image:
+```
+# sbt docker:publishLocal
+```
+Run the image with the environment variables defined either in a file:
+```
+# docker run -it --env-file <.env file> <image ID>
+```
+or explicitly, if variables have been exported to a local environment:
+```
+# docker run -it -e AUTH_URL -e CLIENT_ID -e CLIENT_SECRET -e PROJECT_KEY <image ID>
```

Comment:
Do you mean:
```suggestion
# docker run -it -e AUTH_URL=<auth url> -e CLIENT_ID=<client id> -e CLIENT_SECRET=<client secret> -e PROJECT_KEY=<project key> <image ID>
```
?

Reply:
it works both ways, either to define them within the command or just use the exported ones

Reply:
Oh I see.
Then it should be:
```
AUTH_URL=<auth rul>
CLIENT_ID=<client id>
CLIENT_SECRET=<client secret>
PROJECT_KEY=<project key>
docker run -it -e AUTH_URL -e CLIENT_ID -e CLIENT_SECRET -e PROJECT_KEY <image ID>
```
?

Reply:
ok, will change

Reply:
done